web_help/static/src/trip.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<templates>
+    <t t-name="web_help.TripStep" owl="1">
+        <div>
+            <div class="pb-4">
+                <t t-raw="content" />
+            </div>
+
+            <button
+                class="btn btn-lg btn-outline-light web_help_cb_button mr8"
+            >Got it</button>
+            <button
+                class="btn btn-lg btn-outline-light web_help_close pull-right"
+            >Close</button>
```

Comment:
```suggestion
            >Cancel</button>
```

Nitpicking + Not blocking : I find that "Close" can mean "close the current step", while "Cancel" is more explicit.